src/content/about/index.md

```diff
@@ -2,7 +2,6 @@
 title: About Us
 description: About the team, community and mission of ethereum.org
 lang: en
-sidebar: true
```

Comment:
Not a hard requirement but while we're refactoring all these pages (to largely remove this frontmatter prop), I request we consider renaming it given the component is now the `TableOfContents`. I think table of contents is more obvious/descriptive.

Perhaps we rename this frontmatter prop to be `hideTableOfContents`?

Reply:
We also hide the Edit button here: https://ethereum.org/en/community/events/>

Is this something we might want to do elsewhere too?